src/js/techreport/index.js

```diff
@@ -217,6 +226,38 @@ class TechReport {
     });
   }
 
+  // Get the information about the selected technology
+  getTechInfo() {
+    const technologies = this.filters.app;
+    const technology = technologies.join('%2C')
+      .replaceAll(" ", "%20");
+
+    const base = 'https://prod-gw-2vzgiib6.ue.gateway.dev/v1';
```

Comment:
I see this is defined in a few places. In case we ever need to update the API host, could this be pulled up into a high-level constant?